### ISSUE TITLE
CQLTranslator ignores @Enumerated annotation

### ISSUE THREAD

**Issue by dtuzikov** [Impetus/kundera#231]
Hi!

CQLTranslator ignores annotation @Enumerated(EnumType.STRING/EnumType.ORDINAL)  when building query.

Kundera 2.4

**Comment by mevivs** [comment]
I guess it should work for "EnumType.STRING" ?
Anyway i will get it fixed.
-Vivek

**Comment by dtuzikov** [comment]
No, it doesn't work for both EnumType's

**Comment by xamry** [comment]
Duplicate of #240 ?

**Comment by dtuzikov** [comment]
No, with CQLTranslator its not working even for persisting.

**Comment by pires** [comment]
+1

**Comment by xamry** [comment]
This has been fixed. Please verify in the trunk and close.

**Comment by dtuzikov** [comment]
I have found some problem when using EntityManager.refresh() method, see code below.

``` java
public enum UserSys {
    SYSTEM_A, SYSTEM_B
}
@Embeddable
public class UserDataPK implements Serializable {
    @Column(name = "user_key")
    private String userKey;

    @Column(name = "user_sys")
    @Enumerated(EnumType.STRING)
    private UserSys userSys;

    //... constructor, getters, setters 
}

@Entity
@Table(name = "user_data", schema = "sch001@cassandra_pu")
public class UserData {
    @EmbeddedId
    protected UserDataPK pk;

    @Column (name = "some_data")
    private String someData;

    //... constructor, getters, setters 
}

public class Test  {
    @PersistenceContext(type = PersistenceContextType.EXTENDED)
    private EntityManager em;

    @Test
    public void test1() {
        UserData userData=new UserData();
        userData.setPk(new UserDataPK("test-user-key-1",UserSys.SYSTEM_A));
        userData.setSomeData("abcd");
        em.persist(userData);
        assertNotNull(userData.getPk().getUserSys());
        em.refresh(userData);
        assertNotNull(userData.getPk().getUserSys());
        userData=em.find(UserData.class,new UserDataPK("test-user-key-1",UserSys.SYSTEM_A));
        assertNotNull(userData.getSomeData());
        assertNotNull(userData.getPk().getUserSys()); //<-- FAILS
    }
}
```

log message before fail:
com.impetus.client.cassandra.datahandler.CassandraDataHandlerBase setFieldValueViaCQL WARN: java.io.StreamCorruptedException: invalid stream header: 534F435F

**Comment by mevivs** [comment]
Please verify with 2.5 release.

-Vivek

**Comment by dtuzikov** [comment]
fixed in 2.5
